action-api/src/main/java/com/powsybl/action/DanglingLineAction.java

```diff
@@ -44,4 +42,13 @@ public String getDanglingLineId() {
     public String getType() {
         return NAME;
     }
+
+    @Override
+    public DanglingLineActionBuilder convertToBuilder() {
+        return new DanglingLineActionBuilder().withId(id)
+            .withDanglingLineId(danglingLineId)
+            .withActivePowerValue(activePowerValue)
+            .withRelativeValue(relativeValue)
+            .withReactivePowerValue(reactivePowerValue);
```

Comment:
You can use the get methods instead of using the variables directly (see LoadAction)